src/transactions/test/SetTrustLineFlagsTests.cpp

```diff
@@ -443,8 +443,8 @@ TEST_CASE("revoke from pool",
         }
     };
 
-    auto depositIntoPool = [&](TestAccount& account, Asset const& assetA,
-                               Asset const& assetB) {
+    auto depositIntoPool1 = [&](TestAccount& account, Asset const& assetA,
```

Comment:
Can we give these more descriptive names? 1 and 2 doesn't really tell me anything about what they do/why they're different.